### DISCUSSION TITLE
Delete Camera

### DISCUSSION THREAD

**Discussion by ignacio-db** [aaronwmorris/indi-allsky#1323] [Q&A]
Is there a way to delete a previously configured camera? I have tried different cameras, and now have settled with a particularly one, and would like to clean-up the configuration and free some memory space.

Thanks.

**Comment by aaronwmorris** [comment]
This page has instructions for hiding a camera on the web interface:
https://github.com/aaronwmorris/indi-allsky/wiki/SQL-queries

If you want to delete the files, do the following _before_ hiding the camera.  Select the camera you want to delete in the web interface, and then navigate to System -> Utilities.  From here, you can flush images and timelapses for the camera you want to remove.   **MAKE SURE YOU HAVE THE CORRECT CAMERA SELECTED**

**Reply by ignacio-db** [reply]
Thanks!
